src/bin/util/test/mldsa44-import-key-test.sh

```
@@ -0,0 +1,169 @@
+else
+  cat > "$SOFTHSM2_CONF" <<EOF
+directories.tokendir = $TOKEN_DIR
+objectstore.backend = file
+slots.removable = false
+slots.mechanisms = ALL
+log.level = DEBUG
+log.file = $TOKEN_DIR/token.log
+EOF
+fi
+
+cat $SOFTHSM2_CONF
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)